test/components/canReceiveAndGiveItems.ts

```diff
@@ -0,0 +1,99 @@
+import { IPokemon } from "../../src/components/Battles";
+import { IItemProbabilities } from "../../src/components/mods/RandomHeldItemsMod";
+import { IInventoryListing } from "../../src/components/menus/Items";
+import { FullScreenPokemon } from "../../src/FullScreenPokemon";
+import { it } from "../main";
+import { stubBlankGame } from "../utils/fakes";
+
+it("Pokemon can be given an item and item is updated in inventory", (): void => {
```

Comment:
Filenames typically match up with the component they're testing. Change this one to.